fixtures/builder.py

```diff
@@ -40,6 +41,13 @@ def __init__(self, name):
 
         self.repository.status()
 
+    def __del__(self):
+        # Create a hash for the generated fixture.
+        file_contents = "hash:" + dirhash(self.dir, 'md5')
```

Comment:
Is the word `hash:` really necessary? Also, MD5 isn't supposed to be used any more, since it's vulnerable to rare collisions. Can we use SHA-256?

Reply:
fixed

Reply:
Just to note. I thought that maybe we should use "md5" because it is faster. It is not collision resistant but I think is for deliberate attack. So someone could find a collision if the used a program to brute force and try to find a collision. The chance that we would produce 2 versions of a fixture folder that would *happen* to produce the same hash *and* those produced fixtures would *also* be valid json and pass tests would probably be impossible.

but I ran timer test on `generate_fixtures.py` I saw no consistent difference, probably because producing the hashes are a very small part of the total script time.